.github/workflows/go.yml

```diff
@@ -9,10 +9,10 @@ jobs:
     runs-on: ubuntu-latest
     steps:
 
-    - name: Set up Go 1.17
+    - name: Set up Go 1.18
       uses: actions/setup-go@v2
       with:
-        go-version: 1.17.8
+        go-version: 1.18.2
```

Comment:
Do we want to specify exact version here and just 1.18 in Dockerfile? IMO, it's enough to set 1.18 everywhere

Reply:
change it to v3 and use 1.18.2 I was like to be specific that it must be >= 1.18.2